app/controllers/Auth.scala

```diff
@@ -30,7 +30,6 @@ final class Auth(
       Ok {
         env.user.jsonView.full(
           u,
-          withOnline = true,
```

Comment:
This is the only place except the profile API endpoint where it's true. I guess this is one of those endpoints that will only ever be called with the user being online? Not sure though whether it should return `online -> true` to undo any breakage?